app/components/molecules/SelectorGroup.tsx

```diff
@@ -3,10 +3,13 @@ import { useState } from 'react'
 
 interface SelectorGroupProps {
 	options: { value: string; label: string }[]
+	name: string
+	initialValue?: string
 }
 
-export default function SelectorGroup({ options }: SelectorGroupProps) {
-	let [selectedValue, setSelectedValue] = useState(options[0].value)
+export default function SelectorGroup({ options name,
```

Comment:
You've forgotten a comma between your two function parameters here, and this is causing an error. It should be:

```js
options, name,
```